DEPS

```diff
@@ -132,6 +132,13 @@ hooks = [
                '--source-dir', '.',
                '--filter', '^[0-9]\{{1,\}}\.[0-9]\{{1,\}}\.[0-9]\{{1,\}}$'],
   },
+  {
+    # Downloads & overwrites Chromium's swift-format dep on macOS only
+    'name': 'download_swift_format',
+    'pattern': '.',
+    'condition': 'host_os == "mac"',
```

Comment:
@goodov This matches [Chromium's swift-format DEP](https://source.chromium.org/chromium/chromium/src/+/main:DEPS;l=1778) but we can technically change this to include a `checkout_ios` condition if we want (and then move the script from `build/apple` to `build/ios`), let me know what you prefer!

Reply:
> This matches Chromium's swift-format DEP

that's good. Let's keep it this way to be in sync with Chromium.